storage/migrations/01_oasis_3_consensus.up.sql

```diff
@@ -239,7 +239,7 @@ CREATE TABLE oasis_3.proposals
 
   created_at    UINT63 NOT NULL,  -- EpochTime, i.e. number of epochs since base epoch
   closes_at     UINT63 NOT NULL,  -- EpochTime, i.e. number of epochs since base epoch
-  invalid_votes UINT_NUMERIC NOT NULL DEFAULT 0
+  invalid_votes UINT63 NOT NULL DEFAULT 0
```

Comment:
https://github.com/oasisprotocol/oasis-core/blob/go/v0.2201.11/go/governance/api/proposal.go#L102

Reply:
Well spotted.

When changing the DB types, I wasn't consistent in what I did with values that are uint64 in go. I mapped them to:
- `UINT63` for values that are sequential (nonces, IDs, etc) or have a known range that is unlikely to be >2^63 even in the future (some error codes somewhere IIRC; or maybe that was about uint32)
- `UINT_NUMERIC` where the value is less constrained and I was afraid it might realistically reach the 2^63..2^64 range. This field, and to a lesser extent `max_gas` below, fall into this category.

Perhaps we should leave a comment, something like
```
-- uint64 in go; because the value might conceivably be >2^63, we use UINT_NUMERIC over UINT63 here.
```

Reply:
Got it, updated w/ comment!